ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue

```diff
@@ -124,6 +124,11 @@ export default defineComponent({
         return [HomeLocationTypes.OTHER_STRATA, HomeLocationTypes.OTHER_TYPE]
           .includes(getMhrRegistrationLocation.value.otherType)
       }),
+      hasAddress: computed((): boolean => {
+        return getMhrRegistrationLocation.value.address.street ||
+        getMhrRegistrationLocation.value.address.streetAdditional ||
+        getMhrRegistrationLocation.value.address.city
+      }),
```

Comment:
With longer accessors its always a good idea to optional change, in a computed value like this at least. 
if `address` is ever undefined, it will result in a type error, so an `?` after `address` would protect the conditions.

Reply:
missed that